CAIPs/caip-275.md

```diff
@@ -186,11 +187,13 @@ To further clarify different authentication flow configurations we can consider
 }
 ```
 
-As this authentication flow calls for the `injected` extension, without specifying which one, it should use EIP-6963 to check if a matching extension provider is available, and use that if it is available. If one is not found, we resolve to the default one.
-For ethereum addresses, this would be `window.ethereum`.
-However, this configuration is not recommended as having multiple wallets generates a race condition and can provide false positives where there is a wallet extension but it does not have the account linked to the domain. We recommend EIP-6963 if possible.
+As this authentication flow calls for the `injected` extension, without specifying which one, it should use the default wallet provider at the browser.
+For EVM chains, this would be `window.ethereum`.
 
-#### Wallet connect to any wallet
+However, this configuration is not recommended as having multiple wallets in the browser generates a race condition, and can result in false positives where there is a wallet extension, but it does not have the account linked to the domain.
+We recommend using [EIP-6963][] by specifying a wallet rnds in the URI field if possible, to avoid this issue.
```

Comment:
Maybe it would be smoother to write this section as a 

SHOULD check rdns against 6963 signaling channel first to avoid false-positives or wallet impersonation via protocol pollution

followed by

MUST check window.ethereum if 6963 signal not detected.

This way your own SDK could do it right with 1193 as fallback and lazy implementations or implementations biased towards certain wallets/formfactors would have the option to skimp, but a stronger warning not to?